examples/observe_qt.py

```diff
@@ -0,0 +1,81 @@
+"""
+PyQt example that shows how to utilize observ
+in combination with Qt.
+
+State is passed to the widgets. One of the widgets
+adjusts the state while the other watches the state
+and updates the label whenever a computed property
+based on the state changes.
+"""
+
+from observ import computed, observe, watch
+from PyQt5.QtWidgets import QApplication, QLabel, QPushButton, QVBoxLayout, QWidget
+
+
+class Display(QWidget):
+    def __init__(self, state, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.state = state
+
+        self.label = QLabel()
+
+        layout = QVBoxLayout()
+        layout.addWidget(self.label)
+
+        self.setLayout(layout)
+
+        @computed
+        def label_text():
+            if state["clicked"] == 0:
+                return "Please click the button below"
+            return f"Clicked {state['clicked']} times!"
+
+        self.watcher = watch(lambda: label_text(), self.update_label, immediate=True)
```

Comment:
I guess the API could be simplified a bit so you can write `watch(label_text, ...` 🤔

Reply:
I didn't realize it already supported this syntax! Awesome! <3